utils/codepointtrie/src/codepointtrie.rs

```diff
@@ -104,14 +105,19 @@ fn maybe_filter_value<T: TrieValue>(value: T, trie_null_value: T, null_value: T)
 /// For more information:
 /// - [ICU Site design doc](http://site.icu-project.org/design/struct/utrie)
 /// - [ICU User Guide section on Properties lookup](https://unicode-org.github.io/icu/userguide/strings/properties.html#lookup)
-#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+// serde impls in crate::serde
```

Comment:
Nit: Obsolete comment, and change this so that there is no diff in the serde derive (put on the correct line)